week-3/reading-list/script.js

```diff
@@ -21,3 +21,38 @@ const books = [
   },
 ];
 
+function renderBooks(book) {
```

Comment:
As this function renders only one book I would suggest renaming it to `renderBook` to avoid confusion.